### PR TITLE
added class files

### PR DESCRIPTION
**Pull request by SnehaTimancherla**
check i added .class files

### CODE REVIEW

File: pullrequest [hunk 1]
```diff
@@ -1,2 +1,3 @@
 pull request
 hello
+java.class
```
Comment (sneha818) [inline pullrequest:3]: need to improve code

Comment (SnehaTimancherla) [reply pullrequest:3]: ok i will add more .class files

Comment (sneha818) [reply pullrequest:3]: ok

Comment (sneha818) [inline pullrequest:3]: ok for now  I am adding

**Review by sneha818 — CHANGES_REQUESTED** [review]

**Pull request merged**